docs/Toy/examples/ast.toy

```diff
@@ -1,4 +1,4 @@
-# RUN: python -m toy %s --emit=ast | filecheck %s
+# RUN: python -m toy %s --emit=ast | filecheck %s --strict-whitespace --match-full-lines
```

Comment:
If you hide whitespace changes in the PR viewer you'll see that this is the only change to the file remaining.